modules/ppcp-button/resources/js/modules/ActionHandler/CheckoutActionHandler.js

```diff
@@ -48,7 +48,13 @@ class CheckoutActionHandler {
 
     configuration() {
         const spinner = this.spinner;
-        const createOrder = (data, actions) => {
+        const createOrder = async (data, actions) => {
+            try {
+                await validateCheckoutForm(this.config);
```

Comment:
hm, is it not working for ACDC when doing it like before during the request?

I think this handler is also executed on the pay for order pages while before we were doing this validation only in checkout. Does it cause any issues?

Reply:
Yes, it had the side effect of Pay for Order wanting to validate non present fields.

Previously the `created order` request wasn't validating ACDC and created the PayPal Order. But eventually the `checkout` redirect would detect the missing fields in the form and show the errors and skip the WooCommerce order creation.

I reverted back to validating only on the `create order` endpoint and adding support for credit card validation on that request.